jobs/geo/write_building_footprints.py

```diff
@@ -28,10 +31,23 @@ def write_building_footprints(conn):
     counties = gdf.county_fips.unique()
 
     for x in counties:
-        gdf[gdf.county_fips == x].to_parquet(f"test_{x}.parquet")
+        gdf[gdf.county_fips == x].to_parquet(f"county_fips_{x}.parquet")
         gdf[gdf.county_fips == x].to_file(
-            f"test_{x}.shp"
-        )  # this currently outputs .cpg, .dbf, .shx files in addition to .shp, not sure if that is intended
+            f"county_fips_{x}.shp", driver="ESRI Shapefile"
+        )
+
+        with ZipFile(f"county_fips_{x}_shape_files.zip", "w") as zipped:
+            zipped.write(f"county_fips_{x}.cpg")
+            os.remove(f"county_fips_{x}.cpg")
+
+            zipped.write(f"county_fips_{x}.dbf")
+            os.remove(f"county_fips_{x}.dbf")
+
+            zipped.write(f"county_fips_{x}.shp")
+            os.remove(f"county_fips_{x}.shp")
+
+            zipped.write(f"county_fips_{x}.shx")
+            os.remove(f"county_fips_{x}.shx")
```

Comment:
This is a fine way to create a zipfile normally, but OGR supports writing them natively, which is likely less error-prone, and fewer lines of code. Try writing to a filename ending in `.shp.zip`.